src/util/commandline/app-command.ts

```diff
@@ -11,6 +10,7 @@ export class AppCommand extends Command {
     super(args);
   }
 
+  @shortName("a")
```

Comment:
We'll need to verify this doesn't overlap with anyone else's existing options. Short options are a valuable commodity. That being said, this is a great candidate for one.